### DISCUSSION TITLE
Welcome to systray-x Discussions!

### DISCUSSION THREAD

**Discussion by Ximi1970** [Ximi1970/systray-x#95] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Chlaca** [comment]
It doesn't work for me. I have the tray icon but :
![image](https://github.com/Ximi1970/systray-x/assets/100475365/01e463d2-ed05-4e2c-8d97-ff8943f39b87)
When I try and close Thunderbird, nothing append. Thunderbird is still open.
When I right-click on the tray icon and click on show/hide nothing append...

**Reply by Ximi1970** [reply]
Please disable wayland...
/etc/gdm3/custom.cfg

See updated instructions for xUbuntu.

**Reply by Chlaca** [reply]
Thanks a lot
